frontend/src/views/Layout/ContentFooter.vue

```diff
@@ -15,6 +15,9 @@
           <a href="https://github.com/gradido/gradido/releases/latest" target="_blank">
             App version {{ version }}
           </a>
+          <a :href="'https://github.com/gradido/gradido/commit/' + hash" target="_blank">
```

Comment:
I suggest a `v-if="hast !=== '000000000'"` here to not display this link when it is set to `'000000000'`. If not, we have a link to 404

Reply:
good point